acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -84,10 +84,8 @@ def get_pcp_broker_status(host)
 
 # Start an eventmachine reactor in a thread
 # @return Thread object where the EM reactor should be running
-def start_eventmachine_thread
+def assert_eventmachine_thread_running
```

Comment:
The method name may be confusing. But so could the old name.

PCP-298 is a ticket to refactor these helper libs - I'll add a comment specifying that this EventMachine stuff should be contained within a pcp-client helper lib